lib/Module/IntrinsicCleaner.cpp

```diff
@@ -324,6 +325,19 @@ bool IntrinsicCleanerPass::runOnBasicBlock(BasicBlock &b, Module &M) {
         dirty = true;
         break;
       }
+      case Intrinsic::is_constant: {
+        if(auto* constant = llvm::ConstantFoldInstruction(ii, ii->getModule()->getDataLayout())) {
+          ii->replaceAllUsesWith(constant);
+          ii->eraseFromParent();
```

Comment:
move `ii->eraseFromParent()` outside of the braces as it is a common instruction for both cases. With the single command, you can remove the braces.